src/ImageSharp/Processing/Processors/CloningImageProcessor{TPixel}.cs

```diff
@@ -45,16 +47,12 @@ protected CloningImageProcessor(Image<TPixel> source, Rectangle sourceRectangle)
         protected Configuration Configuration { get; }
 
         /// <inheritdoc/>
-        public Image<TPixel> CloneAndApply()
+        public Image<TPixel> CloneAndExecute()
         {
             try
             {
                 Image<TPixel> clone = this.CreateDestination();
```

Comment:
All `CreateDestination` implementations share the same implementation actually, doing deep clone in fact.

Reply:
Yeah, pretty much. I want to do some refactoring to allow us to do a deep but empty clone without having to write so much code.

Reply:
Or just query the size from the cloning processor, and do the rest in a helper method ;)

Reply:
Made the method private and used the size as an abstract method.